editor/export/editor_export_plugin.h

```diff
@@ -160,6 +161,7 @@ class EditorExportPlugin : public RefCounted {
 	virtual void _get_export_options(const Ref<EditorExportPlatform> &p_export_platform, List<EditorExportPlatform::ExportOption> *r_options) const;
 	virtual Dictionary _get_export_options_overrides(const Ref<EditorExportPlatform> &p_export_platform) const;
 	virtual bool _should_update_export_options(const Ref<EditorExportPlatform> &p_export_platform) const;
+	virtual bool _get_export_option_visibility(const Ref<EditorExportPlatform> &p_export_platform, const String &p_option_name, bool advanced_options_enabled) const;
```

Comment:
I think that rather than having an `advanced_options_enabled` argument, developers should use `get_export_preset()->are_advanced_options_enabled()`.

In implementations of `EditorExportPlatform::get_export_option_visibility()`, we are already getting the advanced flag this way, so this would make the API in `EditorExportPlugin`s match.